book/src/development/adding_lints.md

```diff
@@ -126,10 +126,13 @@ fn main() {
 ```
 
 Note that we are adding comment annotations with the name of our lint to mark
-lines where we expect an error. Once we have implemented our lint we can run
-`TESTNAME=foo_functions cargo uibless` to generate the `.stderr` file. If our
-lint makes use of structured suggestions then this command will also generate
-the corresponding `.fixed` file.
+lines where we expect an error. Except for very specific situations, at least
```

Comment:
```suggestion
lines where we expect an error. Except for very specific situations (`//@check-pass`), at least
```

Or just mention the thing which allows that in parentheses. Then someone knows how to mark them and also what that means, if they encounter it.